src/lua/stack.cr

```diff
@@ -136,6 +136,7 @@ module Lua
     # ```
     #
     def to_s(io : IO)
+      return "" if size == 0
```

Comment:
Is this some bugfix or also related to crystal 0.31 compatibility?

Reply:
It's bug but it's related to Crystal version. See https://travis-ci.org/veelenga/lua.cr/jobs/601296370 - it throws `Arithmetic overflow (OverflowError)
from src/lua/stack.cr:0:29 in 'to_s'`.
If i try code `puts Math.log10(0)` from stack.cr to_s at https://play.crystal-lang.org/ it returns -Infinity for version  0.31.1 but -2147483648 for Crystal 0.27.2 .